Libraries/Database/SessionExtensions.cs

```diff
@@ -89,6 +90,27 @@ public static class SessionExtensions
                 messagesList);
         }
 
+        /// <summary>
+        /// Updates the Content value of a <see cref="Document"/> directly in the Document table. It won't alter the
+        /// <see cref="Document"/>'s version and won't execute index provider's either. Should be used for maintenance
+        /// purposes only.
+        /// </summary>
+        /// <param name="documentId">ID of the <see cref="Document"/> in the Document table.</param>
+        /// <param name="entity">Object that needs to be serialized to the Content field of the Document table.</param>
+        /// <returns><see langword="true" /> if the query updated an existing <see cref="Document"/> successfully.</returns>
+        public static async Task<bool> UpdateDocumentDirectlyAsync(this ISession session, int documentId, object entity)
+        {
+            var transaction = await session.DemandAsync();
+            var dialect = session.Store.Dialect;
+            var content = session.Store.Configuration.ContentSerializer.Serialize(entity);
+
+            var sql = @$"UPDATE {dialect.QuoteForTableName(session.Store.Configuration.TablePrefix + Store.DocumentTable)}
+                SET {dialect.QuoteForColumnName("Content")} = @Content
+                WHERE {dialect.QuoteForColumnName("Id")} = @Id";
+
+            return await transaction.Connection.ExecuteAsync(sql, new { Id = documentId, Content = content }, transaction) > 0;

```

Comment:
Használhatná a `RawExecuteNonQueryAsync()`-et.

Reply:
Azt azért nem tudtam használni, mert nem támogat paramétereket. Paraméterek pedig azért kellenek, mert nem egyértelmű, hogy hogyan lesz beleszerializálva az SQL Query-be pl. a JSON, ez viszont azt megoldja belül valahogy.

Reply:
Ja aha.